infra/modules/providers/azure/container-registry/main.tf

```diff
@@ -0,0 +1,11 @@
+data "azurerm_resource_group" "acr" {
+  name = "${var.resource_group_name}"
+}
+
+resource "azurerm_container_registry" "acr" {
+  name                     = "${var.acr_name}"
+  resource_group_name      = "${data.azurerm_resource_group.acr.name}"
+  location                 = "${data.azurerm_resource_group.acr.location}"
+  sku                      = "${var.acr_sku}"
+  admin_enabled            = "${var.acr_admin_enabled}"
```

Comment:
you're missing `tags` ie `acr_tags`.

Reply:
Same as above, I noticed that and resolved it in a push earlier today